coremltools/test/api/test_api_examples.py

```diff
@@ -731,6 +731,34 @@ def test_unsanitized_input_name_during_prediction():
         error_str = str(error_info.value)
         assert "does not match any of the model input" in error_str
 
+    @staticmethod
+    @pytest.mark.parametrize(
+        "to_tensor", [torch.tensor, tf.convert_to_tensor, lambda x: x.tolist()])
+    @pytest.mark.skipif(not ct.utils._is_macos(), reason="Platform is not Mac OS")
+    def test_torch_tensor_input_during_prediction(to_tensor):
```

Comment:
@fukatani 
since you are testing both tf/ torch input,
would you mind change the test name to something like:
`test_variant_input_type_prediction`?